cyral/assets/configuration/spec.yaml

```diff
@@ -0,0 +1,13 @@
+name: Cyral
+files:
+- name: cyral.yaml
+  options:
+  - template: init_config
+    options:
+    - template: init_config/default
+  - template: instances
+    options:
+    - template: instances/openmetrics
+      overrides:
+        prometheus_url.example: http://localhost:9018/metrics
+    - template: instances/default
```

Comment:
```suggestion
```

Looks like openmetrics template already extends the default one, we won't need to explicitly list it.